README.md

```diff
@@ -14,8 +14,17 @@ Currently there are **2** winning solutions, but I plan to add many more questio
 
 You can also run **Choose Your Own Adventure Game** in Gitpod, a free online dev environment for GitHub.
 
-Please make a **PR** if you want to add a **question, fix grammar/typos, etc.** (will add **CONTRIBUTING GUIDELINES soon**).
```

Comment:
Please link the contributing guidelines.

Reply:
Please review line 28 of the readme file, where I have added the contributing guideline link.

Reply:
Sorry, my bad.

Reply:
No problem :)